lib/Rex/CLI.pm

```diff
@@ -451,8 +451,10 @@ CHECK_OVERWRITE: {
     my %real_groups;
 
     for my $group ( keys %groups ) {
-      my @servers = map { $_->get_servers }
+      my @servers = 
+        map { $_->get_servers }
         Rex::Group->get_group_object($group)->get_servers;
+      @servers = map { $_->name } @servers if $opts{'y'} >= 2;
```

Comment:
I guess the reason I didn't do this is because I don't know how to describe it.  `-Ty` is `List all known tasks, in YAML format`.  But that description applies perfectly to `-Tyy` as well.  

Btw thats the main reason I'm not thrilled with this solution.  I would have preferred to add a key to the output in `-Ty`.

But this is fine too.  Do you have a suggestion for a description?